index.html

```diff
@@ -43,127 +50,162 @@ <h5>
   </section>
   <section class="work">
     <div class="card">
-      <img src="./pictures/Snapshoot-Portfolio0.png" alt="" srcset="">
-      <h2>Tonic</h2>
-      <div class="title">
-        <h5>CANOPY</h5>
-        <ul class="title_list">
-          <li>Back End Dev</li>
-          <li>2015</li>
-        </ul>
+      <img src="./pictures/SnapshootPortfolio.png" alt="" srcset="">
+      <div class="text_side">
+              <h2>Tonic</h2>
+              <div class="title">
+                <h5>CANOPY</h5>
+                <ul class="title_list">
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ul>
+              </div>
```

Comment:
- [ ] We need to stick to the Figma design as much as possible. If you notice, `title` class has the elements that are overlapping each other. This makes the portfolio less user-friendly. I suggest that you verify the padding, margin, or width to ensure that the elements are not overlapping in all the screen sizes.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/220851261-55ef3bb0-06c1-4a00-8710-6694e5ad2e6a.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/220851561-22c70465-1ba4-4dc8-8901-1d44ac4249fa.png)